src/main/java/at/overflow/bukkit/matrixbridge/BridgeService.java

```diff
@@ -35,9 +37,10 @@ private void sendAllPlayers() {
         this.receiver.send("", players.toString());
     }
 
-    public BridgeService(Endpoint receiver, BridgePropertyReader properties) {
+    public BridgeService(Endpoint receiver, JavaPlugin p, BridgePropertyReader properties) {
 
         this.properties = properties;
+	this.plugin = p;
```

Comment:
It's probably better to use just one argument using the `MatrixPlugin` type instead of two parameters that have different static types but contain the same object.